generate-repo-cache.nu

```diff
@@ -12,8 +12,51 @@ def get-metadata [
     |from yaml
 }
 
+def required-attributes [] {
```

Comment:
this could be a simple variable, no need for a function

Reply:
I'd keep it this way. They're used just once, so in-function `let` makes no sense. They should be either *module-scope constants* or just unnamed, according to our stateless standards.

Reply:
Their values might be useful in the future in some different fuctions.